addons/dark-www/classes.js

```diff
@@ -1,8 +1,17 @@
 export default async function ({ addon, console }) {
-  addon.tab.waitForElement(".annual-report-content .covid-response-section").then(() => {
-    document.querySelector(".page").classList.add("sa-annual-report-2019");
+  addon.tab.waitForElement(".annual-report-content > .covid-response-section").then(() => {
+    document.body.classList.add("sa-annual-report-2019");
   });
-  addon.tab.waitForElement(".annual-report-content .looking-forward").then(() => {
-    document.querySelector(".page").classList.add("sa-annual-report-2020");
+  addon.tab.waitForElement(".annual-report-content > .directors-message").then(() => {
+    document.body.classList.add("sa-annual-report-2020");
   });
+  addon.tab.waitForElement(".annual-report-content > .reach-section.t").then(() => {
+    document.body.classList.add("sa-annual-report-2021");
+  });
+  while (true) {
+    const tweet = await addon.tab.waitForElement(".twitter-tweet iframe[src*='theme=light']", { markAsSeen: true });
+    setInterval(() => {
+      tweet.src = tweet.src.replace("theme=light", "theme=dark");
+    }, 0);
```

Comment:
Did you mean `setTimeout`?

By the way, a timeout of 0 can probably be replaced with `queueMicrotask()` and it would avoid mistakes like this.